index.rst

```diff
@@ -683,14 +827,60 @@ Non-configurable CSCs will have no data in the configuration column of the `Mast
 A non-configurable CSC will ignore the ``configurationOverride`` parameter of the ``start`` command, as it has no meaning.
 Likewise these CSCs will not output any of the configuration-related events.
 
+Appendix II: System Requirements
+================================
+
+These are the collection of requirement documents and the requirements that drives the discussion of this tech-note.
```

Comment:
Is there a clearer way to write this? I find "drives the discussion" really hard to understand.

Perhaps something along the lines of "This document derives its requirements from the following documents." (I'm not thrilled with that, either, but in some way we need to explain that how this document traces its requirements back to more fundamental documents).

Reply:
I took a whack at it.